tools/xcms/xcms_plot_eic.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="xcms_plot_eic" name="xcms plot eic" version="@TOOL_VERSION@+galaxy0" profile="21.09">
+    <description>Plot the extracted ion chromatogram (EIC) from mzML file</description>
+    <macros>
+        <import>macros.xml</import>
```

Comment:
Should be `macros_xcms_plot.xml`, isn't it?

Reply:
yes